pom.xml

```
@@ -634,6 +641,7 @@
                     <excludes>
                         <exclude>**/*jmhTest*.java</exclude>
                         <exclude>**/*jmhType*.java</exclude>
+                        <exclude>TestTPCHWithSnowflake.java</exclude>
```

```suggestion
                        <exclude>TestTPCHWithSnowflake.java</exclude>
                        <exclude>TestTPCHWithDuckDB.java</exclude>

```
I think we don't need to run tpch with duckdb everytime. It's too expensive.

---

They both should be tested in every PR. Unfortunately, we are unable to pay for Snowflake.
But the DuckDB is free and we should keep testing to ensure the sqlglot work. It only takes 11 sec.